Src/BootCamp.Chapter/Player.cs

```diff
@@ -75,32 +79,46 @@ public Item[] GetItems(string name)
         // Implement equiping logic and total defense/attack calculation.
         public void Equip(Headpiece head)
         {
-
+            _equipment.Head = head;
```

Comment:
You've done most of the challenge except the last piece. 

"Player cannot equip items that exceed total weight allowed. The total max weight player has is determined by their strength status (30 + strength * 10 (kg))."

Meaning you need to do a check to see if you can equip new items or not based on current total weight.

Reply:
I managed to come up with a fix for it, however it is very verbose and I'm wondering if there is a better way to do so so I'll leave this open for now.